src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -16068,18 +16068,21 @@ public static Vector512<T> Invoke(Vector512<T> x)
 
             public static Vector128<float> Invoke(Vector128<float> x)
             {
-                Vector128<uint> uxMasked = Vector128.Abs(x).AsUInt32();
-                if (Vector128.GreaterThanAny(uxMasked, Vector128.Create(MaxVectorizedValue)))
+                Vector128<float> uxMasked = Vector128.Abs(x);
```

Comment:
What does "masked" denote in this context?

Reply:
Masking off the sign bit. It's just using the name from the amd code.  I've been trying to limit how much this diverges from that code.  But I can change it if it's confusing.

Reply:
It's probably fine given that the amd implementation is linked.